### PR TITLE
{bio}[GCC/11.2.0] Bio-DB-HTS v3.01

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3116.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/9f1e03106fe7e170b21b64680878ec78 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3506.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/9324edfb53314e0feca1231e046cda15 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (3 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3690eed267bde84d432a8153211246a9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0898d10f2748c3cefc3850d16e50cc68 for a full test report.

File: easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+checksums = ['12a6bc1f579513cac8b9167cce4e363655cc8eba26b7d9fe1170dfe95e044f42']
+
+builddependencies = [('pkgconf', '1.8.0')]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('BioPerl', '1.7.8'),
+    ('HTSlib', '1.14'),
+]
+
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: @boegel please restore the fix added to the older version in https://github.com/easybuilders/easybuild-easyconfigs/pull/12253

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: Nice catch, that's also missing in `Bio-DB-HTS-3.01-GCC-10.2.0.eb` (which is what I started from), so I'll add it there too

Comment (boegel) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: fixed in 41a7e5dae4

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node2671.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/9f064b7df5cd6ad21ac6feabf6cc55f6 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0366252f09f0ee00927fdba185834980 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/3a47bec78aded313080b4d762a999b2f for a full test report.

File: easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-GCC-11.2.0.eb:22]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**